# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/text/reftest.list
# Commit: 431c5142ff3a
# Full Hash: 431c5142ff3ad102328d90cb29470b20ccd1271e
# Author: Glenn Watson <github@intuitionlibrary.com>
# Date: 2019-06-13 21:50:57
# Regressor Bug: 1558106
# File Overlap Count: 1
# Description:
#   Bug 1558106 - Support picture caching for multiple scroll roots. r=kvark
#   
#   This patch implements the majority of the planned picture caching
#   improvements. It supports most of the functionality required to
#   (as a follow up) support OS compositor integration. It also improves
# ==============================================================================

diff -r 855d557b82a2 -r 431c5142ff3a gfx/wr/wrench/reftests/text/reftest.list
--- a/gfx/wr/wrench/reftests/text/reftest.list	Wed Jun 12 16:18:22 2019 +0000
+++ b/gfx/wr/wrench/reftests/text/reftest.list	Thu Jun 13 04:43:56 2019 +0000
@@ -7,7 +7,7 @@
 != shadow-cover-2.yaml blank.yaml
 
 skip_on(android) == shadow.yaml shadow-ref.yaml  # Fails on Android emulator and Pixel2
-== shadow-huge.yaml shadow-huge-ref.yaml
+skip_on(android,emulator) == shadow-huge.yaml shadow-huge-ref.yaml
 != shadow-cover-1.yaml shadow-cover-2.yaml
 != shadow-many.yaml shadow.yaml
 != shadow-complex.yaml shadow-many.yaml
@@ -16,7 +16,7 @@
 == decorations.yaml decorations-ref.yaml
 skip_on(android) fuzzy(1,173) == decorations-suite.yaml decorations-suite.png  # Fails on Android emulator and Pixel2
 == 1658.yaml 1658-ref.yaml
-skip_on(android,debug,emulator) fuzzy(1,5) == split-batch.yaml split-batch-ref.yaml  # Android emulator: GL error 502 at blit_framebuffer
+skip_on(android,emulator) fuzzy(1,5) == split-batch.yaml split-batch-ref.yaml  # Android emulator: GL error 502 at blit_framebuffer
 # Next 3 tests affected by bug 1548099 on Android
 skip_on(android) == shadow-red.yaml shadow-red-ref.yaml
 skip_on(android) fuzzy(1,735) == shadow-grey.yaml shadow-grey-ref.yaml
@@ -32,14 +32,14 @@
 skip_on(android) != synthetic-italics.yaml synthetic-italics-ref.yaml
 skip_on(android) != synthetic-italics-custom.yaml synthetic-italics-ref.yaml
 skip_on(android) != synthetic-italics-custom.yaml synthetic-italics.yaml
-options(disable-aa) == ahem.yaml ahem-ref.yaml
+skip_on(android,emulator) options(disable-aa) == ahem.yaml ahem-ref.yaml
 platform(linux) == isolated-text.yaml isolated-text.png
 platform(mac) skip_on(mac,>=10.14) fuzzy(3,67) == white-opacity.yaml white-opacity.png
 fuzzy(1,4) platform(linux) options(disable-subpixel) == colors.yaml colors-alpha.png
 # Run without dual-source blending path, batches are broken when text colors change.
-fuzzy(1,6) platform(linux) options(disable-dual-source-blending) draw_calls(5) == colors.yaml colors-subpx.png
+fuzzy(1,6) platform(linux) options(disable-dual-source-blending) draw_calls(6) == colors.yaml colors-subpx.png
 # Run with both dual-source blending, ensuring batching is improved.
-fuzzy(1,6) platform(linux) draw_calls(2) == colors.yaml colors-subpx.png
+fuzzy(1,6) platform(linux) draw_calls(3) == colors.yaml colors-subpx.png
 platform(linux) options(disable-subpixel) == border-radius.yaml border-radius-alpha.png
 platform(linux) == border-radius.yaml border-radius-subpx.png
 options(disable-aa) == transparent-no-aa.yaml transparent-no-aa-ref.yaml